Sources/IBLinterKit/Config/UseTraitCollectionsConfig.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  UseTraitCollectionsConfig.swift
+//  IBLinterKit
+//
+//  Created by Blazej SLEBODA on 13/09/2020
+//
+
+import Foundation
+
+public struct UseTraitCollectionsConfig: Codable {
+    public let useTraitCollections: Bool
```

Comment:
```suggestion
    public let enabled: Bool
```

How about renaming like this? It would reduce naming redundancy.

Reply:
Done and I am fine with `enabled`.